tutor/src/models/course/builder-ux.js

```diff
@@ -178,7 +178,7 @@ export default class CourseBuilderUX extends BaseModel {
 
   // skips
   skip_new_or_copy() {
-    return !!this.source || Courses.nonPreview.isEmpty;
+    return !!this.source || Courses.isEmpty || Courses.nonPreview.isEmpty;
   }
```

Comment:
ah, but `Courses.isEmpty` could be `false` which would then check the last thing